### DISCUSSION TITLE
Skyline emulator

### DISCUSSION THREAD

**Discussion by S76042** [skyline-emu/skyline#1803] [General]
Plz solve this issues my device is realme 8

**Comment by s3lverd** [comment]
what issue also the skyline discord server is better for support https://discord.gg/skyline-545842171459272705

**Reply by Jamesho27** [reply]
me too have a problem, skyline emu can run for mediatek dimensity 700?

**Reply by Jamesho27** [reply]
please answer....🙏

**Comment by umar248** [comment]
I am using snapdragon 732g and i don't know ehich driver patch should i download for this device to get most out my redmi note 10 pro please tell me which should i download

**Reply by PlayGame666** [reply]
I have the same phone, so I would not refuse to know too
